### PR TITLE
Add f-alert component

### PR DESCRIPTION
**Pull request by pearofducks**

### CODE REVIEW

File: components/alert/f-alert.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              <svg v-else-if="warning" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 17"> <path fill="#D5840B" d="m.2 13 6-11a2.1 2.1 0 0 1 3.6 0l6 11a2 2 0 0 1-1.9 3H2.1a2 2 0 0 1-1.9-3Z"/> <path fill="#fff" fill-rule="evenodd" d="M8 4.3c.4 0 .8.3.8.7v5a.8.8 0 0 1-1.6 0V5c0-.4.4-.8.8-.8Z" clip-rule="evenodd"/> <path fill="#fff" d="M8.8 12.8a.8.8 0 1 1-1.6 0 .8.8 0 0 1 1.6 0Z"/> </svg>
+              <svg v-else xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 17">
+                <!-- nb: logic controlling fill -->
+                <circle cx="8" cy="9" r="8" :fill="fillVar"/>
+                <path fill="#fff" fill-rule="evenodd" d="M7.3 13a.8.8 0 0 0 1.5 0V9a.8.8 0 0 0-1.6 0v4ZM8 5a1 1 0 1 0 0 2 1 1 0 0 0 0-2Z" clip-rule="evenodd"/>
+              </svg>
```
Comment (BalbinaK) [inline components/alert/f-alert.vue:15]: Is there a reason why you don't use icons available in `fabric-ds/design`?

Comment (pearofducks) [reply components/alert/f-alert.vue:15]: At least one of these isn't in the icons repo. We can discuss why or if it should be with Adi maybe!

Comment (BalbinaK) [reply components/alert/f-alert.vue:15]: Should we ask @adidick here? 😄  Which icon is missing?

**Comment by pearofducks** [comment]
I think we can just get the React version done and then fix later? Unless
there's some reason that'd be a blocker?

**Pull request closed without merging**